spec/mailers/prison_mailer_spec.rb

```diff
@@ -94,12 +94,14 @@
     end
 
     it "sends an cancellation notification to a prison so it can be removed from NOMIS" do
+      sample_visit.vo_number = '87654321'
```

Comment:
Might be good if this was moved into the :sample_visit so the VO number appears in mailer previews.